src/python/zquantum/optimizers/simple_gradient_descent.py

```diff
@@ -55,6 +55,8 @@ def _minimize(
         for _ in range(self.number_of_iterations):
             gradients = cost_function.gradient(current_parameters)
             current_parameters = current_parameters - (self.learning_rate * gradients)
+            if keep_history == True:
```

Comment:
the CI pipeline doesn't like this line, i think it just needs to be `if keep_history:`

Reply:
```suggestion
            if keep_history:
```